src/app/contextBroker/contextBroker.cpp

```diff
@@ -1182,6 +1182,13 @@ void orionExit(int code, const std::string& reason)
 */
```

Comment:
Sure, I agree. After all, the behavior of the “black box“ (the broker) has somehow changed ...